src/datepicker/datepicker.component.spec.ts

```diff
@@ -41,7 +43,8 @@ describe("DatePicker", () => {
 			imports: [
 				Calendar16Module,
 				FormsModule
-			]
+			],
+			providers: [ElementService, AnimationFrameServiceSingleton]
```

Comment:
You should be able to just import the `UtilsModule` from `utils/` and just use that :+1: